src/gatling/resources/bodies/fpl/FPLAllocationProposalAdd.json

```diff
@@ -0,0 +1,22 @@
+{
+  "data": {
+     "allocationProposal": {
+        "proposal": "District judge",
+        "proposalReason": "Perf testing reason"
+     }
+  },
+  "event": {
+     "id": "otherProposal",
+     "summary": "",
+     "description": ""
+  },
+  "event_data": {
+     "allocationProposal": {
+        "proposal": "District judge",
+        "proposalReason": "Perf testing reason"
+     }
+  },
+  "event_token": "${existing_case_event_token}",
```

Comment:
I think we should use the same parameter name as the other services for consistency